package.json

```diff
@@ -86,8 +86,8 @@
     "lint:production:stylelint": "tsc --noCheck --outDir ./dist && stylelint '**/*.styles.ts' --custom-syntax postcss-lit --no-color",
     "test": "per-env",
     "test:development": "npm-run-all --parallel test:development:web-test-runner test:development:web-test-runner:serve-coverage",
-    "test:development:playwright": "chokidar './src/*test.visuals*' --initial --silent --command 'playwright test --ignore-snapshots --quiet --only-changed'",
-    "test:development:playwright:comment:chokidar": "`--ui` gives us watching but doesn't allow `--only-changed`. And we need the latter flag so testing is fast locally.",
+    "test:development:playwright": "rimraf ./dist/playwright-baseline-screenshots && chokidar './src/*test.visuals*' --initial --silent --command 'playwright test --only-changed --quiet --update-snapshots all'",
```

Comment:
I found it handy to clear everything out with every run so the directory isn't filled with screenshots that are no longer associated with a test.